### DISCUSSION TITLE
the installations file stops at extracting data/End2.phd

### DISCUSSION THREAD

**Discussion by jd7-art** [LostArtefacts/TRX#957] [Q&A]
the installations file stops at extracting data/End2.phd I can still see the game file and play it but I just wanted to know if I have to also download any thing else

**Comment by walkawayy** [comment]
I haven't seen this issue myself or heard any other reports. Did you somehow have the file open in another program? Maybe try re-running it?

**Comment by aredfan** [comment]
I'm gonna close this because it's quite an old bug report.  If the issue persists, please open a ticket in the issues section instead.
